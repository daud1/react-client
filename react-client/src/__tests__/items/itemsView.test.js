import React from 'react';
import { shallow, render, mount } from 'enzyme';
import ItemsView from '../../components/items/itemsView.component';
import shallowToJson from 'enzyme-to-json';

describe ('ItemView Component', () => {
    it('renders without crashing', () => {
        const wrapper = shallow(<ItemsView />);
        expect(wrapper.exists(<div></div>)).toBe(true)
    });

    it('matches snapshot', () => {
        const wrapper = shallow(<ItemsView />);
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
})