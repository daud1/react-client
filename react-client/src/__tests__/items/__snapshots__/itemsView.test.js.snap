// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemView Component matches snapshot 1`] = `
<div>
  <Search
    setValue={[Function]}
  />
  <withRouter(LogOut) />
  <CreateItem
    callback={[Function]}
    list_id={1}
  />
  <Lister
    callback={[Function]}
  />
</div>
`;
